css/index.css

```diff
@@ -100,18 +132,51 @@ p {
     flex-wrap: wrap;
     justify-content: space-evenly;
 }
+    
 
```

Comment:
These children here inherit almost all styling they need and the children only need to change one style and that is color.  Good job doing this!